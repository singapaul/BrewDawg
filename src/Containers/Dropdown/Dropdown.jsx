import React from "react";
import "./Dropdown.scss";


const Dropdown = ({ label, value, options, onChange }) => {
  return (
    <>

      <label class="label">
        <select value={value} onChange={onChange}>
          {options.map((option) => (
            <option key={option.label} value={option.value}>{option.label}</option>
          ))}
        </select>
      </label>
    </>
  );
};

export default Dropdown;
