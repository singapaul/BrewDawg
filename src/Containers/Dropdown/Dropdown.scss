select{width: 100%;}

h5{padding-bottom: 5px;}
