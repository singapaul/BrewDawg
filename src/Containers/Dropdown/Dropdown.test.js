import { getElementError, render, screen } from "@testing-library/react";
import Dropdown from "./Dropdown";


// render returns object with a container
it("should match form component snapshot", () => {
  const { container } = render(<Dropdown label={"label"} value={"1"} options={["1","2","3"]} />);
  expect(container).toMatchSnapshot();
});
