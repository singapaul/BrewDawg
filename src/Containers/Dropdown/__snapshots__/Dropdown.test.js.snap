// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match form component snapshot 1`] = `
<div>
  <label>
    <select>
      <option />
      <option />
      <option />
    </select>
  </label>
</div>
`;
